src/EdFi.AnalyticsMiddleTier.DataStandard32/Base/PostgreSQL/0020-View-StudentSectionDim-DescriptorDescriptionAsSubject.sql

```diff
@@ -17,7 +17,7 @@ AS
       -- Unfortunately this means that the Staff and StaffSectionAssociation
       -- LastModifiedDate values can't be used to calculate this record's LastModifiedDate
       COALESCE((SELECT
-          STRING_AGG(COALESCE(Staff.FirstName, '') || ' ' || COALESCE(Staff.LastSurname, ''), ',')
+          STRING_AGG(COALESCE(Staff.FirstName, '') || ' ' || COALESCE(Staff.LastSurname, ''), ', ')
```

Comment:
Since this was an existing file that could have been deployed already, should this change be in a new file and not modify an existing view file?